web/docs/guides/database/tables.mdx

```diff
@@ -340,6 +340,12 @@ create table salaries (
 
 A View is a convenient shortcut to a query. Creating a view does not involve new tables or data. When run, an underlying query is executed, returning its results to the user.
 
+:::caution
+
+PostgreSQL views do not support Row Level Security. This will be released in PostgreSQL v15. Only expose views to your API if the data is considered public.
```

Comment:
> PostgreSQL views do not support Row Level Security

The above is not true. I've been trying to refute that myth in many places.  

Maybe something like:

```suggestion
By default, PostgreSQL views bypass Row Level Security unless you change their owner(see https://github.com/supabase/supabase/discussions/1501). PostgreSQL v15 will have a more intuitive control for this.
```